apps/examples/st_things/Kconfig

```diff
@@ -52,6 +52,13 @@ config RESET_BUTTON_PIN_NUMBER
 		The default is the reset button pin number 44 for the ARTIK 053.
 		If you use another board, you can set the pin number for the board.
 
+config EXAMPLE_ST_THINGS_SAMPLE_LIGHT_HANDLE_GPIO_NUM
```

Comment:
Name looks not good. 
1. I don't know what this is by name.  What's the meaning of LIGHT_HANDLE_GPIO?
2. Others have EXAMPLES. Why is only this EXAMPLE?
3. Why do you add SAMPLE? Another is not a sample but this is?